MPU6050/temperature.sh

```diff
@@ -19,3 +19,13 @@ echo "Value of WHO_AM_I register is : $who_am_i"
 echo "Turning on MPU6050"
 i2cset -y 0 0x68 107 0
 
+while true
+do
+TempH=$(i2cget -y 0 0x68 65)
+TempL=$(i2cget -y 0 0x68 66)
+Temp_abs=$(( -(((TempH << 8)+ TempL) ^ ((1 << 0x10) - 1)) ))
```

Comment:
Is this conversation correct for positive values?

Reply:
It is not correct variant, but I fixed script and drivers errors? please, can you check please @yekovalyov

Reply:
Yes. This conversation is correct.